### PR TITLE
Improve export debug info

### PR DESCRIPTION
**Pull request by RDIL**
Improves the export debug info tool to include more data.

### CODE REVIEW

**Review by AnthonyFuller — APPROVED** [review]

File: components/tools.ts [hunk 1]
```diff
@@ -112,10 +120,27 @@ async function exportDebugInfo(): Promise<void> {
...
+    await writeFile(zipFile, makeEmptyArchive())
+
+    const zip = new ZipFS(zipFile, { create: true })
+
+    await zip.writeFilePromise(zip.resolve("meta.json" as Filename), debugJson)
+
+    await copyIntoZip(zip, "logs")
+    await copyIntoZip(zip, "userdata")
+    await copyIntoZip(zip, "contractSessions")
+    await copyIntoZip(zip, "contracts")
```
Comment (moonysolari) [inline components/tools.ts:137]: I don't think `contracts` is really needed?

**Review by moonysolari — COMMENTED** [review]
Maybe also contain all the plugins? Either the `*.js` files or just a list of the plugin names. Might need to look at both the `plugins` folder and under the main Peacock folder.

**Comment by RDIL** [comment]
It contains the plugin names in meta.json

File: components/tools.ts (hunk 1 above)
Comment (RDIL) [reply components/tools.ts:137]: In case of a contract specific problem? Not sure.

**Review by moonysolari — APPROVED** [review]

**Pull request merged**